modules/ROOT/pages/configure-horizontal-autoscaling.adoc

```diff
@@ -0,0 +1,240 @@
+These flags ensure that the cluster autoscaler balances the number of nodes across availability zones. 
+
+For reference on managed EKS cluster, the following node labels are different across node groups:
+
+----
+k8s.amazonaws.com/eniConfigCustom
+vpc.amazonaws.com/eniConfig
+----
+
+Therefore, the cluster autoscaler starts with:
+
+----
+- --balancing-ignore-label=k8s.amazonaws.com/eniConfigCustom
+- --balancing-ignore-label=vpc.amazonaws.com/eniConfig
+----
+
+=== Configure Overprovisioning Pod
+
+Configure at least one overprovisioning pod per availability zone. Each overprovisioning pod holds a resource that can accommodate three Mule app pods, which improves the performance when new nodes are needed.
+
+
+
+[NOTE]
+Refer to your managed vendor documentation for cluster autoscaling best practices and over-provisioning configuration details.
+
+
+=== Configure Topology Spread Constraints
+
+Your Mule app deployments which are enabled with autoscaling, are required to customize the kubernetes resources for the topology spread constraints in pod spec by updating the *whenUnsatisfiable* field with `DoNotSchedule`.
+This configuration ensures that autoscaling replicas are evenly spread across zones for high availability.
+
+----
+topologySpreadConstraints:
+  - maxSkew: 1
+    topologyKey: topology.kubernetes.io/zone
+    whenUnsatisfiable: DoNotSchedule
+    labelSelector:
+      matchLabels:
+        rtf.mulesoft.com/id: "{{ .Values.id }}"
+----
+
+For further details on the configuration, refer to xref:customize-kubernetes-crd.adoc[] documentation.
+
+[WARNING]
+Customizing topology spread with `whenUnsatisfiable: DoNotSchedule`, without the recommended configurations for node groups per availability zone, cluster autoscaler, and overprovisionining, can lead to K8s failing to schedule the replicas with `Pending state` error.
+
+=== Understand CPU-based Autoscaling Policy
+
+MuleSoft owns and applies the https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/[autoscaling^] policy for your Mule application deployments.
+
+The CPU-based HPA policy used for all Mule apps deployed on Runtime Fabric instances, is as follows:
+
+----
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: my-app
+  namespace: app-namespace
+spec:
+  behavior:
+    scaleDown:
+      policies:
+      - periodSeconds: 15
+        type: Percent
+        value: 100
+      selectPolicy: Max
+      stabilizationWindowSeconds: 300
+    scaleUp:
+      policies:
+      - periodSeconds: 180
+        type: Percent
+        value: 100
+      selectPolicy: Max
+      stabilizationWindowSeconds: 0
+  maxReplicas: 3
+  metrics:
+  - resource:
+      name: cpu
+      target:
+        averageUtilization: 70
+        type: Utilization
+    type: Resource
+  minReplicas: 1
```

Comment:
```suggestion
  maxReplicas: <maxReplicasChosenFromUI>
  metrics:
  - resource:
      name: cpu
      target:
        averageUtilization: 70
        type: Utilization
    type: Resource
  minReplicas: <maxReplicasChosenFromUI>
```
min and max replicas are values that the customer can choose from UI, but the text before says that this is choosen by Mule and it is a hard value. We should change it so customers can understand that their values is not 1 and 3 (that 1 and 3 are examples, the same as the app name, but for example what it is under behaviour -> scaleUp and scaleDown, these are all fixed and owned by Mule).

Used this <maxReplicasChosenFromUI> but not sure how to reflect that minReplicas and maxReplicas are being set by the customer.